app/src/main/java/com/gh4a/BaseActivity.java

```diff
@@ -112,6 +112,8 @@
     private Animator mHeaderTransition;
     private final Handler mHandler = new Handler();
 
+    private final List<Disposable> mDisposeOnStop = new ArrayList<>();
```

Comment:
Use `CompositeDisposable` and its `dispose` method.

Reply:
Is that one clearable, IOW can one reuse it after calling dispose?